components/Image/style.scss

```diff
@@ -1,10 +1,12 @@
 %img-align {
   &-right {
-    float: right;
+    float: unset;
     margin-left: .75rem;
+    margin-right: 0;
   }
   &-left {
-    float: left;
+    float: unset;
+    margin-left: 0;
     margin-right: .75rem;
   }
   &-center {
```

Comment:
I'd like to be able to remove the below `display: block` but I couldn't figure out what to do with lines 38-42 in that case:
```
&[width="80%"],
    &[align=center],
    &[alt~=align-center] {
      @extend %img-align-center;
    }
```

So I just left it alone!